.github/workflows/run_periodic_tests.yml

```diff
@@ -38,42 +38,49 @@ jobs:
           python -m pip install pre-commit
           pre-commit run -a
```

Comment:
The style job has to be removed (we don't want it altogether, not just not needing it for other jobs)

Reply:
You can also use the "Add suggestion to batch" or "Commit suggestion" buttons going forward because they make the job easier for you when there are a lot of review comments